src/hub.js

```diff
@@ -541,6 +541,15 @@ const onReady = async () => {
   const initialEnvironmentEl = document.createElement("a-entity");
   initialEnvironmentEl.addEventListener("bundleloaded", () => {
     remountUI({ initialEnvironmentLoaded: true });
+
+    for (const modelEl of initialEnvironmentEl.querySelectorAll(":scope > [gltf-model-plus]")) {
```

Comment:
note this style selector wasn't working in edge in the one other place we do it (of course edge support is not going to happen anytime soon :))